README.md

```diff
@@ -32,6 +32,7 @@ latitudeShApi.Profile.get().then((response) => {
 # Available API Methods
 
 - `ApiVersion.get` Params: `()`. [Reference](https://docs.latitude.sh/reference/get-current-version)
+- `ApiVersion.getAll` Params: `()`. [Reference](https://docs.latitude.sh/reference/get-api-versions)
```

Comment:
How about using ``list`` instead ``getall``. The list is widely used to represent a request that returns all items in the SDK. 
```suggestion
- `ApiVersion.list` Params: `()`. [Reference](https://docs.latitude.sh/reference/get-api-versions)
```